containers/message-parser/app/default_schemas/ecr.json

```diff
@@ -354,6 +354,14 @@
           "category": "RR Reportability Information"
         }
       },
+      "code": {
+        "fhir_path": "iif(Observation.valueCodeableConcept.text.exists(), Observation.valueCodeableConcept.text, Observation.valueCodeableConcept.coding.code)",
```

Comment:
same here

Reply:
though I don't think we use this schema? so you can just leave it alone. Only `core.json` and `extended.json` are in use that I know of